ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/BasicAction.java

```diff
@@ -3386,6 +3386,19 @@ private final boolean checkChildren (boolean isCommit)
                     tsLogger.i18NLogger.warn_coordinator_BasicAction_57(get_uid());
                 } else {
                     tsLogger.i18NLogger.warn_coordinator_BasicAction_58(get_uid());
+
```

Comment:
@mmusgrov comment reminded me of this: https://dmlloyd.github.io/presentations/coding-standards/index.html#/27. I'm just not sure what does he mean by saying "most constructs".